flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -183,6 +184,8 @@
 	/** Registry that tracks state which is shared across (incremental) checkpoints. */
 	private SharedStateRegistry sharedStateRegistry;
 
+	private boolean isPerferCheckpointForRecovery;
```

Comment:
There is a typo in basically every use of the word "preferCheckpointForRecovery" it should be prefer not "perfer". I think the easiest would be to search the changes and correct it in every place.

Reply:
Sorry for the typo, will fix it soon.